File: OLSource1/v1/vs140/codesnippet/VisualBasic/how-to--modify-a-silverlight-screen-by-using-code_1.vb
        Private Sub FindControlInList()
            Dim index As Integer = 0

            For Each cust As Customer In Customers

                If cust.CompanyName = "Great Lakes Food Market" Then
                    With FindControlInCollection("CompanyName", Customers(index))
                        .IsVisible = False
                        .IsReadOnly = True
                    End With

                End If
                index = index + 1
            Next
        End Sub

File: OLSource1/v1/vs140/codesnippet/CSharp/how-to--modify-a-silverlight-screen-by-using-code_1.cs
        private void FindControlInList()
        {
            int index = 0;

            foreach (Customer cust in this.Customers)
            {
                if (cust.CompanyName == "Great Lakes Food Market")
                {
                    this.FindControlInCollection("CompanyName",
                    this.Customers.ElementAt(index)).IsVisible = false;
                    this.FindControlInCollection("CompanyName",
                    this.Customers.ElementAt(index)).IsReadOnly = true;
                }

                index++;
            }

        }